.github/file-filters.yml

```diff
@@ -33,16 +33,20 @@ backend_lintable: &backend_lintable
 backend_dependencies: &backend_dependencies
   - 'requirements-*.txt'
 
+backend_build_changes: &backend_build_changes
+  - 'Makefile'
```

Comment:
My main purpose is that I want backend changes to be triggered when the `Makefile` gets modified.
Let me know what you think.

Reply:
I think this makes sense because it fits in with the other files in this group